cost-analyzer/templates/_helpers.tpl

```diff
@@ -115,6 +115,50 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 app: cost-analyzer
 {{- end -}}
 
+{{/*
+Return the appropriate apiVersion for daemonset.
+*/}}
+{{- define "cost-analyzer.daemonset.apiVersion" -}}
+{{- if semverCompare "<1.9-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
The `GitVersion` changed `Version` between helm 2 & 3

https://v2.helm.sh/docs/chart_template_guide/

https://helm.sh/docs/chart_template_guide/builtin_objects/

See the `Capabilities` section of each

Reply:
These templates are lifted directly from the prometheus chart. I imagine we're going to have more issues there as well? https://github.com/kubecost/cost-analyzer-helm-chart/blob/3fc635168847a0c356c8e46d6a88361665f734d6/cost-analyzer/charts/prometheus/templates/_helpers.tpl#L182-L221

Reply:
Just ran this template with helm 3:

```
kubeVersion: {{ .Capabilities.KubeVersion.GitVersion }}
```

and it generated:
```
kubeVersion: v1.16.0
```

Helm Info:
```bash
~/Repos/kubecost/cost-analyzer-helm-chart$ helm3 version
version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}
```

It appears this is fine, and backwards compatible.

Reply:
iirc correction, this shows up in Minikube with Helm 3